### PR TITLE
Nv32 configuration changes for gst and stress tests

### PR DESCRIPTION
**Pull request by manoj-freyr**

### CODE REVIEW

File: rvs/conf/nv32/gst_stress_3_hrs.conf [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# ################################################################################
+# #
+# # Copyright (c) 2018-2022 Advanced Micro Devices, Inc. All rights reserved.
```
Comment (jkottiku) [inline rvs/conf/nv32/gst_stress_3_hrs.conf:3]: 2023 instead of 2022 Right ?

File: rvs/conf/nv32/gst_single.conf [hunk 2]
```diff
@@ -55,7 +55,7 @@ actions:
   count: 1
   duration: 10000
   copy_matrix: false
-  target_stress: 7000
+  target_stress: 6000
```
Comment (jkottiku) [inline rvs/conf/nv32/gst_single.conf:58]: To be in tune with Navi32 based on trail and error ?

**Review by jkottiku — CHANGES_REQUESTED** [review]

File: rvs/conf/nv32/gst_stress_3_hrs.conf (hunk 1 above)
Comment (manoj-freyr) [reply rvs/conf/nv32/gst_stress_3_hrs.conf:3]: will do, yes

File: rvs/conf/nv32/gst_single.conf (hunk 2 above)
Comment (manoj-freyr) [reply rvs/conf/nv32/gst_single.conf:58]: yes, mostly this was having some issues in SRIOV setup, and all feature enabled for sriov setup is gonna happen in later release. this one is with some trial and error and some calculation thrown in :)

**Review by jkottiku — APPROVED** [review]

**Pull request merged**